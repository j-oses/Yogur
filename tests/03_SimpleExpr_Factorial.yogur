// Un pequeño cálculo del factorial para ir calentando motores

var fact : Int = 1
var n : Int = 5
while n > 1 {
    fact = fact * n
	n=n-1
}