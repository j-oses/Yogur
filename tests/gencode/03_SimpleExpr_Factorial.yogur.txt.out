Run successfully completed
Final P-machine state:
PC = 26   MP = 0   SP = 1   EP = -1   NP = 100   code size = 27
STORE = 
(0,Int 120)
(1,Int 1)
(2,Bool False)
(3,Int 1)
(4,Int 1)
(5,Null)
(6,Null)
(7,Null)
(8,Null)
(9,Null)
(10,Null)
(11,Null)
(12,Null)
(13,Null)
(14,Null)
(15,Null)
(16,Null)
(17,Null)
(18,Null)
(19,Null)
(20,Null)
(21,Null)
(22,Null)
(23,Null)
(24,Null)
(25,Null)
(26,Null)
(27,Null)
(28,Null)
(29,Null)
(30,Null)
(31,Null)
(32,Null)
(33,Null)
(34,Null)
(35,Null)
(36,Null)
(37,Null)
(38,Null)
(39,Null)
(40,Null)
(41,Null)
(42,Null)
(43,Null)
(44,Null)
(45,Null)
(46,Null)
(47,Null)
(48,Null)
(49,Null)
(50,Null)
(51,Null)
(52,Null)
(53,Null)
(54,Null)
(55,Null)
(56,Null)
(57,Null)
(58,Null)
(59,Null)
(60,Null)
(61,Null)
(62,Null)
(63,Null)
(64,Null)
(65,Null)
(66,Null)
(67,Null)
(68,Null)
(69,Null)
(70,Null)
(71,Null)
(72,Null)
(73,Null)
(74,Null)
(75,Null)
(76,Null)
(77,Null)
(78,Null)
(79,Null)
(80,Null)
(81,Null)
(82,Null)
(83,Null)
(84,Null)
(85,Null)
(86,Null)
(87,Null)
(88,Null)
(89,Null)
(90,Null)
(91,Null)
(92,Null)
(93,Null)
(94,Null)
(95,Null)
(96,Null)
(97,Null)
(98,Null)
(99,Null)
